### PR TITLE
DOCS-9641 initial draft content.

### PR DESCRIPTION
**Pull request by sleslie23**

### CODE REVIEW

File: modules/ROOT/pages/runtime-fabric-logs.adoc [hunk 1]
```diff
@@ -274,6 +275,32 @@ Example output plugin configuration:
...
+
+[%header%autowidth.spread,cols="a,a,a,a"]
+.Syslog Configuration Parameters
+|===
+| Key | Description | Required | Default Value
+| Host | Host name or IP address of the remote Syslog server | Yes |
+| Port | TCP or UDP port of the remote Syslog server | Yes | 514
+| Mode | The mode of the transport type | Yes | udp
+| Syslog format | Syslog protocol format | No | rfc5424
+| Syslog Format | Syslog protocal format | No |
```
Comment (sarjeet2013) [inline modules/ROOT/pages/runtime-fabric-logs.adoc]: Duplicate? Same line as above?

File: modules/ROOT/pages/runtime-fabric-logs.adoc [hunk 2]
```diff
@@ -274,6 +275,32 @@ Example output plugin configuration:
...
+[%header%autowidth.spread,cols="a,a,a,a"]
+.Syslog Configuration Parameters
+|===
+| Key | Description | Required | Default Value
+| Host | Host name or IP address of the remote Syslog server | Yes |
+| Port | TCP or UDP port of the remote Syslog server | Yes | 514
+| Mode | The mode of the transport type | Yes | udp
+| Syslog format | Syslog protocol format | No | rfc5424
+| Syslog Format | Syslog protocal format | No | 
+| Syslog Maxsize | Integer value that sets the maximum number of bytes per message. | No | 1024 for rfc3164
```
Comment (sarjeet2013) [inline modules/ROOT/pages/runtime-fabric-logs.adoc]: Comma inbetween 1024 for rfc3164, 2048 for rfc5424?

File: modules/ROOT/pages/runtime-fabric-logs.adoc (hunk 1 above)
Comment (hegderohit89) [reply modules/ROOT/pages/runtime-fabric-logs.adoc]: Good catch! Thanks

**Review by sarjeet2013 — DISMISSED** [review]

File: modules/ROOT/pages/runtime-fabric-logs.adoc (hunk 1 above)
Comment (sleslie23) [reply modules/ROOT/pages/runtime-fabric-logs.adoc]: Sorry, that was from a merge conflict. I assume the former is the one we want to keep?

Comment (sleslie23) [reply modules/ROOT/pages/runtime-fabric-logs.adoc]: Resolved.

File: modules/ROOT/pages/runtime-fabric-logs.adoc [hunk 3]
```diff
@@ -274,6 +275,30 @@ Example output plugin configuration:
...
+
+[%header%autowidth.spread,cols="a,a,a,a"]
+.Syslog Configuration Parameters
+|===
+| Key | Description | Required | Default Value
+| Host | Host name or IP address of the remote Syslog server | Yes |
+| Port | TCP or UDP port of the remote Syslog server | Yes | 514
+| Mode | The mode of the transport type | Yes | udp
+| Syslog format | Syslog protocol format | No | rfc5424
+| Syslog Maxsize | Integer value that sets the maximum number of bytes per message. | No | 1024 for rfc3164, 2048 for rfc5424
```
Comment (dukesphere) [inline modules/ROOT/pages/runtime-fabric-logs.adoc]: the others don't have a period at the end, prob remove it

**Review by dukesphere — DISMISSED** [review]
might want to lose that period (see comment)

File: modules/ROOT/pages/runtime-fabric-logs.adoc (hunk 3 above)
Comment (hegderohit89) [reply modules/ROOT/pages/runtime-fabric-logs.adoc]: Ok i will remove it

**Review by sarjeet2013 — APPROVED** [review]

**Review by dukesphere — APPROVED** [review]

**Pull request merged**